### PR TITLE
Remove UI, Girder, and devops from this repository

### PR DESCRIPTION
**Pull request by manthey**
The UI and devops components have moved to HistomicsUI or digital_slide_archive.

The ctk_cli adjustments are now taken from the girder-slicer-cli-web package.

Switch to using tox instead of the Girder 2/CMake test harness.

Moved from unittest to pytest for some of the tests.

Fix several warnings from the Bugbear warning set, mainly about using mutable default variables in function definitions.

Disable tests that still need to be converted.  These tests should all be converted to run with pytest and use local resources or data files pulled from a standardized source rather than development servers.

### CODE REVIEW

**Comment by manthey** [comment]
@kheffah This removes all of the code that has moved to other repositories.  Some of the tests are now using pytest (the main difference is in `assert`, instead of `assertFunction()` you use `assert <condition>`.  The tests that haven't been converted over have been disabled.

You should be able to run all tests locally with tox: `tox -e py37,flake8` will run the tests against Python 3.7 and check linting, for instance.

**Comment by kheffah** [comment]
@manthey When I try to run the tests with tox:
```bash
tox -e py36,flake8
```
It gets stuck in the dependency installation stage:

```
CMake Error: The current CMakeCache.txt directory /tmp/pip-req-build-is4wlo24/_skbuild/linux-x86_64-3.6/cmake-build/CMakeCache.txt is different than the directory /home/mtageld/Desktop/HistomicsTK/_skbuild/linux-x86_64-3.6/cmake-build where CMakeCache.txt was created. This may result in binaries being created in the wrong place. If you are not sure, reedit the CMakeCache.txt
  CMake Error: The source "/tmp/pip-req-build-is4wlo24/CMakeLists.txt" does not match the source "/home/mtageld/Desktop/HistomicsTK/CMakeLists.txt" used to generate cache.  Re-run cmake with a different source directory.
    File "/tmp/pip-build-env-up18snb_/overlay/lib/python3.6/site-packages/skbuild/setuptools_wrap.py", line 574, in setup
      languages=cmake_languages
    File "/tmp/pip-build-env-up18snb_/overlay/lib/python3.6/site-packages/skbuild/cmaker.py", line 232, in configure
      os.path.abspath(CMAKE_BUILD_DIR())))
```
I have an existing histomicstk installation, can't I use my existing installation when running the test with tox without having this dependency re-installation in a temporary directory? If not, woud editing/deleting CMakeCache.txt to be able to run the test mess up the existing installation in any way?

**Comment by manthey** [comment]
@kheffah Can you revert your last commit (renaming various tests).  Some of those were running successfully, even though they subclasses `unittest`.

**Comment by manthey** [comment]
@kheffah If you approve this PR, I'll merge it in.  We can fix and reenable tests in future PRs.

File: histomicstk/annotations_and_masks/annotations_to_masks_handler.py [hunk 1]
```diff
@@ -8,22 +8,15 @@
```
[the remark this answers is not in the document]
Comment (manthey) [reply histomicstk/annotations_and_masks/annotations_to_masks_handler.py:8]: Restored.  @kheffah Are we good to merge?

**Pull request merged**